sealing.go

```diff
@@ -205,6 +209,19 @@ func (m *Sealing) newSector() (abi.SectorNumber, error) {
 		return 0, xerrors.Errorf("starting the sector fsm: %w", err)
 	}
 
+	sd, err := m.getSealDelay()
+	if err != nil {
+		return 0, xerrors.Errorf("getting the sealing delay: %w", err)
+	}
+
+	if sd > 0 {
+		timer := time.NewTimer(sd)
+		go func() {
+			<-timer.C
```

Comment:
If the sector's state changes (say, for instance, that the miner manually starts sealing the sector), this timer will start fire and trigger the `StartPacking`, right?

That should just fail gracefully, but is icky...Any quick suggestions?

Reply:
So that sends the `SectorStartPacking` event, which is only handled by those 2 states - https://github.com/filecoin-project/storage-fsm/blob/61380dcf4a8b9e731237234fce984af59af85a0c/fsm.go#L39-L44

For states using the planOne function, when they receive an unexpected event they will cause some combustion - https://github.com/filecoin-project/storage-fsm/blob/61380dcf4a8b9e731237234fce984af59af85a0c/fsm.go#L373

I think a decant workaround for that would be to create an interface like `type Ignorable interface {Igronable()}`, put that on the SectorStartPacking evernt, and then ignore in planOne if it didn't match anything

Reply:
Thanks! Implemented and tested this, it works well.